docs/dev/datastore.rst

```diff
@@ -58,122 +57,8 @@ present locally.
-of the environment.
-
-Prepare zenodo_store
-^^^^^^^^^^^^^^^^^^^^
-
-Our `zenodo_store <https://github.com/catalyst-cooperative/pudl-zenodo-storage>`__
-script initializes and updates data sources that we maintain on
-`Zenodo <https://zenodo.org/>`__ . It prepares
-`Frictionless Datapackages <https://frictionlessdata.io/>`__ from scraped files and
-uploads them to the appropriate Zenodo archive.
-
-To add a new archive to our Zenodo storage collection:
-
-* Update ``zs.metadata`` with a UUID and metadata for the new Zenodo archive.
-   These details will be used by Zenodo to identify and describe the archive on
-   the website.  The UUID is used to uniquely distinguish the archive **prior to
-   the creation of a DOI.**
-* Prepare a new library to handle the **frictionless datapackage** descriptor of
-  the archive.
-
-  * The library name should take the form ``frictionless.DATASET_raw``.
-  * The library must contain
-    `frictionless data metadata <https://specs.frictionlessdata.io/data-package/#language>`__
-    describing the archive.
-  * The library must contain a ``datapackager(dfiles)`` function that:
-
-    * receives a list of
-      `zenodo file descriptors <https://developers.zenodo.org/#deposition-files>`__
-    * converts each to an appropriate
-      `frictionless datapackage resource descriptor <https://specs.frictionlessdata.io/data-resource/#language>`__
-    * **Important**: The resource descriptor must include an
-      additional ``descriptor["remote_url"]`` that contains
-      the zenodo url to download its resource.  This will be the same
-      as the ``descriptor["path"]`` at this stage.
-    * If there are criteria by which you wish to be able to discover or
-      filter specific resources, ``descriptor["parts"][...]`` should be
-      used to denote those details.  For example,
-      ``descriptor["parts"]["year"] = 2018`` would be appropriate to
-      allow filtering by year.
-    * Combines the resource descriptors and frictionless metadata to produce
-      the complete datapackage descriptor as a python dict.
-* In the ``bin/zenodo_store.py`` script:
-
-  * Import the new frictionless library.
-  * Add the new source to the ``archive_selection`` function; follow the
-    format of the existing selectors.
-  * Add the new source name to the help text in the ``parse_main() ..
-    deposition`` argument.
-
-The above steps should be sufficient to allow automatic initialization and
-updates of the new data source on Zenodo.
-
-You initialize an archive (preferably starting with the sandbox) by running
-``zenodo_store.py --initialize --verbose --sandbox``
-
-If successful, the DOI and url for your archive will be printed.  You will
-need to visit the url to review and publish the Zenodo archive before it can
-be used.
-
-If you lose track of the DOI, you can look up the archive on Zenodo using the
-UUID from ``zs.metadata``.
-
-Prepare the Datastore
```

Comment:
This section on the `Datastore` should probably remain. the `Datastore` is contained in PUDL, and hasn't changed (yet)